src/main/java/com/robertx22/age_of_exile/database/data/currency/base/GearCurrency.java

```diff
@@ -25,7 +25,7 @@ public abstract class GearCurrency extends Currency {
 
     public abstract int getPotentialLoss();
 
-    public boolean isAffectedByPotential() {
+    public boolean isPotentialConsumer() {
```

Comment:
change the method name for clarity.
Also change the currency desc in Words.java